crates/tests/src/e2e/ibc_tests.rs

```diff
@@ -3218,3 +3288,29 @@ fn packet_forward_memo(
     })
     .expect("Test failed")
 }
+
+fn gen_refund_target(test: &Test, alias: &str) -> Result<()> {
```

Comment:
This should really be a disposable address but we don't have the right support for it in the wallet. I'll add a note to #4040